### PR TITLE
#165412889 & #165412878 Add article rating and article read time

### PR DESCRIPTION
**Pull request by kagaramag**
### What does this PR do?
- Article rating
- Article read time

#### Description of Task to be completed?
- Create an article rating
- Retrieve article rating average
- Display article read time

#### How should this be manually tested?
After cloning the repo, `run` the command `npm run dev`, and navigate to an article of your choice from the home page

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#165412889 
#165412878

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/__tests__/actions/rating/createRate.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'react-thunk';
+import { Provider } from 'react-redux';
+// import article from '../../../__mocks__/article';
```
Comment (MBenedicte) [inline src/__tests__/actions/rating/createRate.test.js]: remove the comment

File: src/__tests__/actions/rating/createRate.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    expect(component).toMatchSnapshot();
+  });
+  const dispatch = jest.fn(action => action);
+
+  describe('Create rating', () => {
+    test('returns rating information', async () => {
+      const result = createRate(rating)(dispatch);
+      expect(result).toHaveProperty('type');
+      expect(result).toHaveProperty('payload');
+      // expect(result.payload).toEqual(rating);
```
Comment (MBenedicte) [inline src/__tests__/actions/rating/createRate.test.js]: the same here

File: src/__tests__/reducers/rating/ratingReducers.test.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import createRatingReducer from '../../../reducers/rating/createRatingReducer';
+import initialState from '../../../store/initialStates/ratingInitialState';
+import { ratingActionsTypes } from '../../../actions-types';
+// import articles from '../../__mocks__/articles';
+// import article from '../../__mocks__/article';
+// import message from '../../__mocks__/articleMessage';
+// import errors from '../../__mocks__/errors';
+
```
Comment (MBenedicte) [inline src/__tests__/reducers/rating/ratingReducers.test.js:4]: Please make sure that all comments are removed

**Review by MBenedicte — CHANGES_REQUESTED** [review]

**Review by sengayire — APPROVED** [review]

File: src/__tests__/reducers/rating/ratingReducers.test.js (hunk 3 above)
Comment (sengayire) [reply src/__tests__/reducers/rating/ratingReducers.test.js:4]: remove comments

**Review by sengayire — APPROVED** [review]

**Review by MBenedicte — APPROVED** [review]

**Pull request merged**